plugins/snortlabslist/unit_test/test_lookup.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase
+from unittest.mock import patch
+from komand_snortlabslist.actions.lookup import Lookup
+
+
+class TestLookup(TestCase):
+    @patch("requests.get")
+    def test_run_found(self, mock_get):
+        mock_response = mock_get.return_value
+        mock_response.text = "1.2.3.4\n5.6.7.8\n"
+        action = Lookup()
+        params = {"address": "1.2.3.4"}
+
+        result = action.run(params)
+
+        self.assertTrue(result["found"])
+        self.assertEqual(result["address"], "1.2.3.4")
+        self.assertEqual(result["status"], "No Error")
+
+    @patch("requests.get")
+    def test_run_not_found(self, mock_get):
+        mock_response = mock_get.return_value
+        mock_response.text = "1.2.3.4\n5.6.7.8\n"
+        action = Lookup()
+        params = {"address": "9.10.11.12"}
+
+        result = action.run(params)
+
+        self.assertFalse(result["found"])
+        self.assertEqual(result["address"], "9.10.11.12")
+        self.assertEqual(result["status"], "No Error")
```

Comment:
Could you use `parameterized` to parameterise those tests? Also, that'd be awesome to move some parts to `def setUp(self) -> None` such as `action` etc 😄